From c0535e09deee6919a3d6d801cbf17757add3d9a4 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Wed, 27 May 2020 17:30:13 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-evb-ind: set vdd_log compatible
 as regulator-fixed

fix:
[    0.454816] of_pwm_get(): can't parse "pwms" property
[    0.455277] pwm-regulator vdd-log: Failed to get PWM: -2
[    0.455762] pwm-regulator: probe of vdd-log failed with error -2

Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
Change-Id: I0ca4f0ab2d2bde6e17507477f8b7c29882edab46
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
index 578c3aaf9fe7..b3ab41606e33 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
@@ -266,16 +266,12 @@
 	};
 
 	vdd_log: vdd-log {
-		compatible = "pwm-regulator";
+		compatible = "regulator-fixed";
 		regulator-name = "vdd_log";
-		regulator-min-microvolt = <800000>;
-		regulator-max-microvolt = <1400000>;
+		regulator-min-microvolt = <950000>;
+		regulator-max-microvolt = <950000>;
 		regulator-always-on;
 		regulator-boot-on;
-
-		/* for rockchip boot on */
-		rockchip,pwm_id= <2>;
-		rockchip,pwm_voltage = <900000>;
 	};
 };
 
-- 
2.35.3

